app/src/main/java/openfoodfacts/github/scrachx/openfood/analytics/AnalyticsView.kt

```diff
@@ -0,0 +1,8 @@
+package openfoodfacts.github.scrachx.openfood.analytics
+
+enum class AnalyticsView(val path: String) {
+    Scanner("scanner"),
```

Comment:
Let's use the common convention of having enums values as UPPERCASE:
https://stackoverflow.com/questions/3069743/coding-conventions-naming-enums

Reply:
np I'll change it, but I think in kotlin we should revisit some coding styles :)
good article https://arturdryomov.dev/posts/kotlin-enum-recipes/

Reply:
I usually follow Android Studio hints, but yeah we should adopt a style.